src/components/event/details/DetailsSuspectedCommits.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <DetailsBase
+    class="details-suspected-commit"
+    :expandShowed="commits.length > 3"
+    @expandClicked="isMoreCommitsShown = !isMoreCommitsShown"
+  >
+    <template #header>
+      SUSPECTED COMMITS
+    </template>
+    <template #content>
+      <div
+        v-for="(commit, index) in filteredCommits"
+        :key="index"
+        class="event-details__content-block details-suspected-commit__content-block"
+        data-ripple
+      >
+        <div class="details-suspected-commit__header-row">
+          <div class="details-suspected-commit__left">
+            <div class="details-suspected-commit__left-title">
+              {{ commit.title }}
+            </div>
+          </div>
+          <div class="details-suspected-commit__right">
+            <div class="details-suspected-commit__right_block">
+              <span class="details-suspected-commit__right_block-commitHash">
+                {{ commit.commitHash.substr(0, 7) }}
+              </span>
+            </div>
+          </div>
+        </div>
```

Comment:
I think it's worth changing the arrangement of the blocks so that the commit hash is centered in the design. I would do something like in the picture.

![изображение](https://user-images.githubusercontent.com/40016207/120048369-94543d80-c01f-11eb-8c0c-7d9c88874457.png)